HW0/P6/P6.txt

```diff
@@ -1 +1,47 @@
 # Your answers here
+
+
+1)
+A typical result I get when I run P6A.py is:
+Hi Job 0
+Hi Job 2
+Hi Job 3
+Hi Job 1
+Bye Job 0
+Hi Job 4
+Bye Job 3
+Hi Job 5
+Bye Job 2
+Bye Job 1
+Hi Job 6
+Hi Job 7
+Bye Job 4
+Hi Job 8
+Bye Job 5
+Hi Job 9
+Bye Job 7
+Bye Job 6
+Bye Job 8
+Bye Job 9
+
+We see that Jobs are not finished in the same order as they are started.
+We also see that jobs are not started in the right order.
+A possible explanation is that there are sometimes some lock in the
```

Comment:
Interesting, in my case the jobs always started in the right order, and all the "Hi"'s were in order throughout, but the Byes came out all kapakahi.